src/json/webhooks/ReferralClassicDisabled.schema.json

```diff
@@ -45,6 +55,11 @@
       "description": "The Id of the program this code was created for"
     }
   },
-  "required": ["id", "referrer", "referred", "dateReferralStarted"],
+  "required": [
+    "id",
+    "referrer",
+    "referred",
```

Comment:
I think we discussed `referred` not being a required field here? Please check.

Reply:
I thought that was only for classic, but ill double check with Scott

Reply:
Just checked, only not required for classic